src/style.css

```diff
@@ -0,0 +1,28 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url("./images/star.svg");
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--5 .stars__star,
```

Comment:
it will be better to add proper `:nth-child` because what if you’ll have more than 5 stars children but the number of active stars will be `5`?